docs/linux/fs.md

```diff
@@ -419,89 +419,91 @@ Birth: -
-   avoid excessive append write by allowing some random writes within segments.
-   The LFS mode forces sequential writes to segments and the sequential use of
-   segments within sections, resulting in full compliance with zoned block
-   devices write constraint.
-3. **Zone reset as discard operation** Block *discard* (or *trim*) used to
-   indicate to a device that a block or range of blocks are no longer in use is
-   replaced with execution of a zone write pointer reset command when all blocks
-   of all segments of a section are free, allowing the section to be reused.
-
-Compared to a solution using the *dm-zoned* device mapper target, performance
-of *f2fs* on zoned devices does not suffer from zone reclaim overhead as writes
-are always sequential and do not require on-disk temporary buffering. *f2fs*
-garbage collection (segment cleanup) will generate overhead only for workloads
-frequently deleting file or modifying files data.
+2. **Forced LFS mode** By default, *f2fs* tries to optimize block allocation (in   order to avoid excessive append write) by allowing some random writes within 
+   segments. The LFS mode forces sequential writes to segments and forces the 
+   sequential use of segments within sections, which results in full compliance 
+   with the zoned block device's write constraint.
+3. **Zone reset as discard operation** In the past, block *discard* (or *trim*) 
+   indicated to a device that a block or range of blocks are no longer in use. 
+   This has been replaced with the execution of a "zone write pointer reset" 
+   command when all blocks of all segments of a section are free. This allows 
+   the section to be reused.
+
+Compared to a solution that uses the *dm-zoned* device mapper target, 
+the performance of *f2fs* on zoned devices does not suffer from "zone reclaim 
+overhead", because writes are always sequential and do not require on-disk 
+temporary buffering. *f2fs* garbage collection (segment cleanup) generates 
+overhead only for workloads that frequently delete files or modify files' data.
 
 ### Zone Capacity Support
 
 NVMe ZNS SSDs can have a per
 [zone capacity that is smaller than the zone size](../introduction/zns#zone-capacity-and-zone-size).
 To support ZNS devices, *f2fs* ensures that block allocation and accounting
-only considers the blocks in a zone that are within the zone capacity. This
-support for NVMe ZNS zone capacity is available since Linux kernel version 5.10.
+considers only the blocks in a zone that are within the zone's capacity. This
+support for NVMe ZNS zone capacity has been available since it was introduced 
+in Linux kernel version 5.10.
 
-Additionally, *f2fs* volumes need some storage space that is randomly writable
-to store and update in-place metadata blocks for the volume. Since NVMe zoned
+*f2fs* volumes need some storage space that is randomly writable in order 
+to store and update in-place metadata blocks for the volume. Since NVMe-zoned
```

Comment:
I would not add the "-" between NVMe and zoned, to keep aligned with the standard wording.